napari/layers/_image_layer/view/properties.py

```diff
@@ -18,8 +18,8 @@ def __init__(self, layer):
         comboBox.setCurrentIndex(index)
         comboBox.activated[str].connect(
             lambda text=comboBox: self.changeColor(text))
-        self.grid_layout.addWidget(QLabel('colormap:'), 3, 0)
-        self.grid_layout.addWidget(comboBox, 3, 1)
+        self.grid_layout.addWidget(QLabel('colormap:'), 3, 0, 1, 2)
+        self.grid_layout.addWidget(comboBox, 3, 2, 1, 2)
```

Comment:
I find all the magic numbers quite offputting. Perhaps they can be defined, or at least used as kwargs so that newcomers can see what they all mean?

Reply:
I think we need a follow up PR fixing this across the whole code base, I'd rather not do this partially here. In the long run I #285 will be part of the solution, but there is lots we could do before then